src/index.js

```diff
@@ -1 +1,80 @@
-console.log("Hello World!");
+import './style.css';
+
+// const gameID = 'X9fxHdLZsiAYP8qUTFeL';
```

Comment:
- [OPTIONAL] Please kindly delete the comment line of code to have a good and clean codebase 🙃 🙃